packages/web-react/src/components/Toast/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const TRANSITION_DURATION = 250;
+
+export const TRANSITIONING_STYLES: Record<string, string> = {
+  entering: 'is-open is-transitioning',
+  entered: 'is-open',
+  exiting: 'is-hidden is-transitioning',
+  exited: 'is-hidden',
+};
+
+export const DEFAULT_ICONS: Record<string, string> = {
```

Comment:
The almost same (without inverted) object is in dictionaries (`EMOTION_COLORS`, I think). And it can be used for establishing this object of the defaults.

Reply:
I don't think it's the same. This is an object with icon names; emotion colors are not connected to icon names.

Reply:
As we discussed in the office, I created a new hook for icon names.